source/ShopPanel.cpp

```diff
@@ -67,8 +67,8 @@ namespace {
 
 	void DrawTooltip(const string &text, const Point &hoverPoint, const Color &textColor, const Color &backColor)
 	{
-		constexpr int WIDTH = 250;
-		constexpr int PAD = 10;
+		const int WIDTH = 250;
+		const int PAD = 10;
```

Comment:
constexpr is for things (usually functions or objects) that can be evaluated at compile time. this does not need to be evaluated, a simple const will suffice (and it fits with all the other code, that just uses const for ints)

Reply:
This is unrelated to the new feature and should rather be handled when implementing #9864.

Reply:
It was related because that was the code I had copied into the other class. I was on the verge of extracting it into a new Tootip class myself, and made the change because I wanted the two blocks of code to be identical so that they could be identified as the same later on. This change is immaterial to the operation of the program, and should not block the PR IMO.